src/pybind/mgr/orchestrator_cli/module.py

```diff
@@ -164,6 +181,55 @@ def _list_services(self, cmd):
 
             return HandleCommandResult(odata="\n".join(lines))
 
+    def _create_osd(self, cmd):
+        """Create one or more OSDs
+
+        :cmd : Arguments for the create osd
+        """
+        #Obtain/validate parameters for the operation
+        cmdline_error = ""
+        if "json" in cmd.keys():
+            params = cmd["json"]
+        elif "file" in cmd.keys():
+            try:
+                with open(cmd["file"]) as f:
```

Comment:
No. This is not going to work. Could you please use the `inbuf` parameter of `handle_command` for directly inserting JSON?

Reply:
Fixed!! Thx!!